Doc/library/sqlite3.rst

```diff
@@ -1087,23 +1087,10 @@ committed:
 .. literalinclude:: ../includes/sqlite3/ctx_manager.py
 
 
-Common issues
--------------
-
-Multithreading
-^^^^^^^^^^^^^^
-
-Older SQLite versions had issues with sharing connections between threads.
-That's why the Python module disallows sharing connections and cursors between
-threads. If you still try to do so, you will get an exception at runtime.
-
-The only exception is calling the :meth:`~Connection.interrupt` method, which
-only makes sense to call from a different thread.
-
 .. rubric:: Footnotes
 
 .. [#f1] The sqlite3 module is not built with loadable extension support by
    default, because some platforms (notably Mac OS X) have SQLite
-   libraries which are compiled without this feature. To get loadable
-   extension support, you must pass --enable-loadable-sqlite-extensions to
-   configure.
+   libraries which are compiled without this feature. To get loadable extension
+   support, your Python must be compiled with
+   ``-–enable-loadable-sqlite-extensions`` option in ``PYTHON_CONFIGURE_OPTS``.
```

Comment:
AFAIK `PYTHON_CONFIGURE_OPTS` is specific to pyvenv. The old sentence was correct. `configure` is the script mentioned in the [build instructions](https://github.com/python/cpython#build-instructions). Please revert it.

Also, we don't usually accept PRs with random formatting changes.